_docs/_latest/setup/forseti-on-gke.md

```diff
@@ -15,8 +15,37 @@ The following tools are required:
 * [Terraform](https://www.terraform.io/downloads.html) - 0.12.x
 * [gsutil](https://cloud.google.com/storage/docs/gsutil)
 
+## Service Account Credentials
+Terraform uses an IAM Service Account to deploy and configure resources on behalf of the user.  This
+service account and key can be created by executing the [setup.sh](https://github.com/forseti-security/terraform-google-forseti/blob/master/helpers/setup.sh)
+helper script.
+
+### Obtain the Setup Script
+The following will download the setup script locally.
+```sh
+curl --location --remote-name https://raw.githubusercontent.com/forseti-security/terraform-google-forseti/master/helpers/setup.sh
+chmod +x setup.sh
+./setup.sh -h
+```
+
+### Run the Setup Script
+Execute the setup script replacing **ORG_ID** and **PROJECT_ID** with the orgainization ID and project ID respectively.
+```sh
+setup.sh -o ORG_ID -p PROJECT_ID -k
+```
+The above command will result in a *credentials.json* being downloaded to the your current working directory.
+
+### Export the Credentials
+Set the appropriate environment variable to the path of the service account JSON
+key file:
+
+```sh
+export GOOGLE_APPLICATION_CREDENTIALS=credentials.json
```

Comment:
FYI - The [setup script](https://github.com/forseti-security/terraform-google-forseti/blob/module-release-5.0.0/helpers/setup.sh#L121) will now export this environment variable. In order for that export to stick around after the script is executed, you need to source it.

Reply:
Ok.  I can take this out.